src/main/java/com/salessparrow/api/config/CoreConstants.java

```diff
@@ -34,7 +34,7 @@ public static String awsRegion() {
   }
 
   public static String cacheClusterId() {
-    return System.getenv("CACHE_CLUSTER_ID");
+    return System.getenv("SALESSPARROW_CACHE_CLUSTER_ID");
```

Comment:
SALESSPARROW_CACHE_CLUSTER_ID it not defined anwhere?

Reply:
Updated the variable name